moped-editor/src/components/GridTable/Filters.js

```diff
@@ -38,6 +38,7 @@ import {
   handleApplyValidation,
   isFilterNullType,
   shouldRenderAutocompleteInput,
+  shouldUseFreeSolo,
```

Comment:
my mind immediately went to https://en.wikipedia.org/wiki/Free_Solo

Reply:
+1 such an incredible doc!